### ISSUE TITLE
Table Completions not returning all tables

### ISSUE THREAD

**Issue by TylerR909** [mtxr/vscode-sqltools#1118]
**Describe the bug**

We evidently have **253 tables** but when I type `SELECT * FROM `, Intellisense is only offering me some of my tables. Per Output debug I can see:

```
[1678406221628] INFO  (ext): EXECUTING COMMAND => sqltools.getConnections
[1678406221629] INFO  (ls): REQUEST RECEIVED => connection/GetConnectionsRequest
[1678406225456] INFO  (ls): completion requested {"line":1,"character":15}
    ns: "intellisense"
[1678406225464] INFO  (ls): got got %d db completions
    ns: "intellisense"
[1678406225500] INFO  (ls): got 100 table completions
    ns: "intellisense"
```

If I type out a table that won't show up in Intellisense, and then continue on to fill out WHERE or SELECT then intellisense _does_ give me proper column intellisense. 

**To Reproduce**
1. Make a couple hundred tables
1. Type `SELECT * FROM ` and prompt Intellisense to suggest a table
1. Not all the tables are there
    1. Results appear to be alpha sorted 

**Expected behavior**
All my tables appear in Intellisense

**Desktop (please complete the following information):**
 - SQLTools Version: v0.27.1
 - VS Code Version: 1.76.0
 - OS: OSX
 - Driver:
   - [ ] PostgreSQL/Redshift
   - [x] PostgreSQL/Cockroach
   - [ ] MySQL/MariaDB
   - [ ] MSSQL/Azure
   - [ ] SQLite
   - [ ] Other? Which...
 - Database version: [e.g. MySQL v5.6, PostgreSQL vX...]
   - PostgreSQL 13.4 (Debian 13.4-4.pgdg110+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit

**Additional context**

I've tried setting `"sqltools.results.limit": 200,` but no joy

**Comment by TylerR909** [comment]
Almost a year later. I just retried with CockroachDB and the Postgres drivers and it still can't intellisense anything past the first 100 tables. I use intellisense to explore and branch out to see what my options are. This is unusable with a limit that low. We're up to 319 tables now describing various entities, many-to-many relationships, Enumeration tables, etc. The inability to see past "E" in an alpha-sorted table intellisense is pretty rough.

**Comment by gjsjohnmurray** [comment]
If you type an initial letter (e.g. T) before invoking Intellisense do you get shown up to 100 choices beginning with that letter?

**Comment by TylerR909** [comment]
Nope. I have a table called `projects` which alphasorted clocks in at 225.

`SELECT * from p` does not offer that table up. It's suggesting tables with `p` in the name, like `activity_type`, but it's acting like it loaded the first 100 tables, cached those, and now Intellisense is only searching those cached tables. I have 31 _other_ tables that look like `project_status` or `project_team_member` etc and none of them are suggested.

**Comment by Felipe-53** [comment]
+1, I can confirm this happens. In my case, with MySQL. It only suggests a few of the tables. Plus, as described in other issues such as this one https://github.com/mtxr/vscode-sqltools/issues/1065  it also fetches tables from all of the databases of the connection, not just from the database selected.

**Comment by Shuangcheng-Ni** [comment]
It seems that [this line](https://github.com/mtxr/vscode-sqltools/blob/d397dc42f685c793db8067d13b2949dc07267002/packages/driver.mysql/src/ls/queries.ts#L138) causes the problem.

**Comment by pavlo-tk** [comment]
In my case, table suggestions are drawn from a completely different database on the same connection. So, essentially, table suggestions don't work.
